### ISSUE TITLE
ordering of columns in a matrix of markers

### ISSUE THREAD

**Issue by lmsimp** [ComputationalProteomicsUnit/pRolocGUI#78]
When executing the below code I find the top GO annotations include things such as proteasome and EIF 3 complex etc. which is what we expect, however when I then run `pRolocVis` to view these results the order of these terms has been changed so it's alphabetical! 

``` r
library("pRoloc")
library("pRolocVis")
library("pRolocdata")
data("hyperLOPIT2015")
params <- setAnnotationParams(inputs = c("Mus musculus",
                                         "UniProt/Swissprot"))
cc <- addGoAnnotations(hyperLOPIT2015, params,
                       namespace = "cellular_component")
cc <- filterMinMarkers(cc)
cc <- filterMaxMarkers(cc)
res <- orderGoAnnotations(cc)

head(colnames(fData(res)$GOAnnotation))
[1] "eukaryotic translation initiation factor 3 complex"
[2] "eukaryotic 43S preinitiation complex"              
[3] "eukaryotic 48S preinitiation complex"              
[4] "proteasome core complex"                           
[5] "cytosolic large ribosomal subunit"                 
[6] "proteasome complex" 

## Does not produce this order when visualised?
pRolocVis(res, fcol = "GOAnnotations")
```

**Comment by lmsimp** [comment]
I think I see what is happening, in `pRolocVis` when the `fcol` is a `matrix` of markers we run `mrkMatToVec` and this converts to a `character` and so the ordering of the markers (column order in the matrix) is lost. 

If I store this order, then after running `mrkMatToVec` make a `factor` and preserve the `levels`. What do you think @lgatto?

**Comment by lmsimp** [comment]
Ignore the above comment... I think I can simplify my code to avoid the above happening in the first places, before I can `mrkVecToMat`

**Comment by lmsimp** [comment]
This has been fixed in commit 9b354ad0722082fb008f1629233cfc430e92cf8e but now I need to fix in the compare app
